app/riot/README.md

```diff
@@ -0,0 +1,26 @@
+# Storybook for Riot <sup>alpha</sup>
+
+* * *
+
+Storybook for Riot is a UI development environment for the components written with riot.js.
+With it, you can visualize different states of your UI components and develop them interactively.
+
+![Storybook Screenshot](https://github.com/storybooks/storybook/blob/master/media/storybook-intro.gif)
+
+Storybook runs outside of your app.
+So you can develop UI components in isolation without worrying about app specific dependencies and requirements.
+
+## Getting Started
+
+```sh
+npm i -g @storybook/cli
+cd my-app
+getstorybook --html
```

Comment:
You should have a `getstorybook` for Riot.

Reply:
Working on that topic currently